docs/quick_start.md

```diff
@@ -160,10 +160,10 @@ Tezos.contract
     println(`Waiting for ${op.hash} to be confirmed...`);
     return op.confirmation(1).then(() => op.hash);
   })
-  .then((hash) => println(`Operation injected: https://edo.tzstats.com/${hash}`))
+  .then((hash) => println(`Operation injected: https://florence.tzstats.com/${hash}`))
   .catch((error) => println(`Error: ${JSON.stringify(error, null, 2)}`));
 ```
 
 [boilerplate]: https://github.com/ecadlabs/taquito-boilerplate
 [smart_contract_source]: https://ide.ligolang.org/p/CelcoaDRK5mLFDmr5rSWug
-[smart_contract_on_better_call_dev]: https://better-call.dev/edo2net/KT1F7DYSa7fVTNScSDDVVokqmmytpJBB5bs9/operations
+[smart_contract_on_better_call_dev]: https://better-call.dev/florencenet/KT1F7DYSa7fVTNScSDDVVokqmmytpJBB5bs9/operations
```

Comment:
I couldn't find this contract on BCD. Screenshot attached - 
![Screen Shot 2021-05-03 at 9 01 11 AM](https://user-images.githubusercontent.com/2622360/116893518-4d288780-abee-11eb-8eca-755f4c133c0e.png)

Reply:
its there for me:

https://better-call.dev/search?text=KT1F7DYSa7fVTNScSDDVVokqmmytpJBB5bs9

Reply:
thanks, the link that shows where the contact can be found on BCD is a link to the wrong contract. I have corrected it.